src/carbons.c

```diff
@@ -152,6 +152,7 @@ void carbons_xml_stripped_cb(PurpleConnection * gc_p, xmlnode ** stanza_pp) {
   char * buddy_name_bare      = (void *) 0;
   PurpleConversation * conv_p = (void *) 0;
   PurpleAccount * acc_p       = (void *) 0;
+  char * body_data;
```

Comment:
could you set this to `NULL` as well so that it's definitely safe to call `g_free()` on it?

Reply:
I addressed this in the last commit